packages/firebase_inappmessaging/lib/firebase_inappmessaging.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+
+class FirebaseInAppMessaging {
+  static const MethodChannel _channel =
+      const MethodChannel('plugins.flutter.io/firebase_inappmessaging');
+
+  static Future<void> triggerEvent(String eventName) async {
```

Comment:
Rather than being `static` this should be an instance method and `FirebaseInAppMessaging` should have an `instance` getter. See cloud_functions for an example https://github.com/flutter/plugins/blob/master/packages/cloud_functions/lib/src/cloud_functions.dart#L28

Reply:
done!